hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -666,13 +672,15 @@
       sw_images: ["//sw/device/tests:otbn_ecdsa_op_irq_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+sw_test_timeout_ns=28_000_000"]
+      run_timeout_mins: 400
```

Comment:
Many thanks @sriyerg for taking care of this!

Regarding the run time of the OTBN ECDSA test: we are aware that many OTBN tests take a very long time to run and I believe that's exactly why most of them are only run on FPGA but not in simulation for CI. I am not sure we really need to run an ECDSA operation for this chip-level test or if a simpler/faster OTBN operation would be sufficient. Looking up the [commit which changed the testplan to use ECDSA](https://github.com/lowRISC/opentitan/commit/484a00fcf5d7201a6d6239b60509918beb9a344d) and the linked meeting notes I couldn't find references regarding why it needs to be an ECDSA test. Do you think it would be okay to switch to something simpler/faster?

Reply:
Absolutely agree. If you notice, the testpoint is generically named as `chip_otbn_op`. It does not matter what operation OTBN executes (its software anyway). IIRC I wired up the ecdsa test to it at the time since that had been developed.

Reply:
Thanks for the feedback @sriyerg . I've filed an issue here to track this: #14385 .